tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -33,6 +34,7 @@
 class CollaborationContext implements Context {
 	private FeatureContext $featureContext;
 	private SpacesContext $spacesContext;
+	private string $lastAppOpenResponse;
```

Comment:
it is actually not a reponse but a response body, so this would be good name
```suggestion
	private string $lastAppOpenData;
```